Tests/Approval.Tests/ApiApproval.cs

```diff
@@ -19,72 +16,34 @@ namespace Approval.Tests;
+
     [Theory]
     [ClassData(typeof(TargetFrameworksTheoryData))]
     public Task ApproveApi(string frameworkVersion)
     {
-        string codeBase = Assembly.GetExecutingAssembly().Location;
-        var uri = new UriBuilder(new Uri(codeBase));
-        string assemblyPath = Uri.UnescapeDataString(uri.Path);
-        var containingDirectory = Path.GetDirectoryName(assemblyPath);
-        var configurationName = new DirectoryInfo(containingDirectory).Parent.Name;
-
-        var assemblyFile = Path.GetFullPath(
-            Path.Combine(
-                GetSourceDirectory(),
-                Path.Combine("..", "..", "Src", "FluentAssertions", "bin", configurationName, frameworkVersion,
-                    "FluentAssertions.dll")));
-
-        var assembly = Assembly.LoadFile(Path.GetFullPath(assemblyFile));
+        var configuration = typeof(ApiApproval).Assembly.GetCustomAttribute<AssemblyConfigurationAttribute>()!.Configuration;
+        var assemblyFile = GetPath("Src", "FluentAssertions", "bin", configuration, frameworkVersion, "FluentAssertions.dll");
+        var assembly = Assembly.LoadFile(assemblyFile);
         var publicApi = assembly.GeneratePublicApi(options: null);
 
         return Verifier
             .Verify(publicApi)
             .ScrubLinesContaining("FrameworkDisplayName")
             .UseDirectory(Path.Combine("ApprovedApi", "FluentAssertions"))
-            .UseStringComparer(OnlyIncludeChanges)
             .UseFileName(frameworkVersion)
             .DisableDiff();
     }
 
-    private static string GetSourceDirectory([CallerFilePath] string path = "") => Path.GetDirectoryName(path);
-
-    // Copied from https://github.com/VerifyTests/Verify.DiffPlex/blob/master/src/Verify.DiffPlex/VerifyDiffPlex.cs
-    public static Task<CompareResult> OnlyIncludeChanges(string received, string verified, IReadOnlyDictionary<string, object> _)
-    {
-        var diff = InlineDiffBuilder.Diff(verified, received);
-
-        var builder = new StringBuilder();
-
-        foreach (var line in diff.Lines)
-        {
-            switch (line.Type)
-            {
-                case ChangeType.Inserted:
-                    builder.Append("+ ");
-                    break;
-                case ChangeType.Deleted:
-                    builder.Append("- ");
-                    break;
-                default:
-                    // omit unchanged files
-                    continue;
-            }
+    private static string GetSolutionDirectory([CallerFilePath] string path = "") => Path.Combine(Path.GetDirectoryName(path)!, "..", "..");
 
-            builder.AppendLine(line.Text);
-        }
-
-        var compareResult = CompareResult.NotEqual(builder.ToString());
-        return Task.FromResult(compareResult);
-    }
+    private static string GetPath(params string[] paths) => Path.GetFullPath(Path.Combine(paths.Prepend(GetSolutionDirectory()).ToArray()));
```

Comment:
🔧 We like to order the methods in calling order, so this method should be positioned at the bottom. 
🤔 I think naming this `CombinedPaths` would better cover its purpose, right?

Reply:
Addressed in 58fbc6de6255e71bf806f708bc4d6bf46d1c388a